stubs/paho-mqtt/paho/mqtt/subscribeoptions.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+from typing_extensions import TypedDict
+
+from . import MQTTException as MQTTException
```

Comment:
Should `MQTTException` really be re-exported from here?

Reply:
It does not pass `stubtest` if left out. So I would let it be.